src/pages/dashboard/menu.js

```diff
@@ -41,7 +48,10 @@ const MenuPage = () => {
     }, [menuId])
 
     const updateMenuSelection = (menu) => {
-        if (typeof menu === 'undefined') {
+        if(menu === 'all-menus') {
+            setMenuId('all-menus')
+        }
+        else if (typeof menu === 'undefined') {
```

Comment:
Just a side comment if we are gonna set the default tab to all menus - we can use 'undefined' for all-menus.

@cchyung Have we decided on that yet ^^^^?

Reply:
I also need to look into what happens when there are no menus at all... not sure how changing this would affect that

Reply:
@clairepo Hmmm yeah in that case I guess there will be one tab (all-menus) that's empty. Maybe just pull the prompt to cr
![Screen Shot 2020-10-26 at 11 58 40 AM](https://user-images.githubusercontent.com/24697473/97132653-994bc880-1782-11eb-8ca7-a87f0f276f7d.png)
eate menus over to the empty tab.